homeassistant/components/usage_prediction/common_control.py

```diff
@@ -55,7 +53,6 @@
     Platform.SENSOR,
     Platform.SIREN,
     Platform.SWITCH,
-    Platform.TEXT,
```

Comment:
Just a note: The UniFi Protect integration provides a text entity for some doorbells. It's used to change the text on the integrated display. So, the text domain can represent some device functionality at least.

Looking at the other integrations providing this domain, it seems to only be used for "virtual functionality" in those, so it's probably fine to just remove this.

Reply:
Yeah, let's see how people use it and move from there. I want it to be mainly about _real_ device controls.